internal/handlers/common/aggregations/stage.go

```diff
@@ -36,14 +36,53 @@ type Stage interface {
 // stages maps all supported aggregation stages.
 var stages = map[string]newStageFunc{
 	// sorted alphabetically
-	"$count": newCount,
-	"$group": newGroup,
-	"$limit": newLimit,
-	"$match": newMatch,
-	"$sort":  newSort,
+	"$count":  newCount,
+	"$group":  newGroup,
+	"$limit":  newLimit,
+	"$match":  newMatch,
+	"$sort":   newSort,
+	"$unwind": newUnwind,
 	// please keep sorted alphabetically
 }
 
+// unsupportedStages maps all unsupported yet stages.
+var unsupportedStages = map[string]struct{}{
+	"$addFields":              {},
+	"$bucket":                 {},
+	"$bucketAuto":             {},
+	"$changeStream":           {},
+	"$collStats":              {},
+	"$currentOp":              {},
+	"$densify":                {},
+	"$documents":              {},
+	"$facet":                  {},
+	"$fill":                   {},
+	"$geoNear":                {},
+	"$graphLookup":            {},
+	"$indexStats":             {},
+	"$limit":                  {},
+	"$listLocalSessions":      {},
+	"$listSessions":           {},
+	"$lookup":                 {},
+	"$merge":                  {},
+	"$out":                    {},
+	"$planCacheStats":         {},
+	"$project":                {},
+	"$redact":                 {},
+	"$replaceRoot":            {},
+	"$replaceWith":            {},
+	"$sample":                 {},
+	"$search":                 {},
+	"$searchMeta":             {},
+	"$set":                    {},
+	"$setWindowFields":        {},
+	"$sharedDataDistribution": {},
+	"$skip":                   {},
+	"$sortByCount":            {},
+	"$unionWith":              {},
+	"$unset":                  {},
```

Comment:
Let's track unsupported stages. Otherwise, we return "not implemented yet" error for every non-existent stage like `$foo`